### PR TITLE
AGR-11 Automated groups reporting

### PR DESCRIPTION
**Pull request by sam-ludlow**
https://io1dev.atlassian.net/browse/AGR-11

### CODE REVIEW

File: api.swagger.yaml [hunk 1]
```diff
@@ -1355,27 +1355,108 @@ paths:
...
             type: string
             enum:
-              - relevance
+              - group_name
               - likes
               - members_count
+              - members_type
+              - post_count
+              - privacy_setting
+              - relevance
```
Comment (ParryLuke) [inline api.swagger.yaml:1456]: default sort should be created date, switch to relevance for searches

File: api.swagger.yaml [hunk 2]
```diff
@@ -1355,27 +1355,108 @@ paths:
...
+            type: array
+            items:
+              type: string
+              enum:
+                - Public
+                - Private
+                - Secret
+        - description: 'Filter groups with the membership types. Multiple values may be supplied.'
+          in: query
+          name: members_type
```
Comment (ParryLuke) [inline api.swagger.yaml:1417]: let's take this out for now, I don't think it's needed

**Review by ParryLuke — CHANGES_REQUESTED** [review]

File: api.swagger.yaml (hunk 1 above)
Comment (sam-ludlow) [reply api.swagger.yaml:1445]: done

File: api.swagger.yaml (hunk 2 above)
Comment (sam-ludlow) [reply api.swagger.yaml]: done

**Review by ParryLuke — APPROVED** [review]

**Review by ParryLuke — APPROVED** [review]

**Review by ParryLuke — APPROVED** [review]

**Review by ParryLuke — APPROVED** [review]

**Review by ParryLuke — APPROVED** [review]

**Comment by MireilleHelle** [comment]
We'll use https://github.com/invotra/api/pull/323 instead. Closing PR.

**Pull request closed without merging**